build.sbt

```diff
@@ -3,20 +3,71 @@ import sbt._
+)
+
+lazy val library = (project in file("."))
   .enablePlugins(SbtAutoBuildPlugin, SbtGitVersioning, SbtArtifactory)
   .settings(
-    majorVersion                     := 10,
-    makePublicallyAvailableOnBintray := true
-  ).settings(
-    scalaVersion        := "2.11.12",
-    crossScalaVersions  := Seq("2.11.12", "2.12.8"),
-    libraryDependencies ++= AppDependencies.compile ++ AppDependencies.test,
-    PlayCrossCompilation.playCrossCompilationSettings,
-    scalacOptions       ++= Seq("-deprecation"),
-    resolvers           := Seq(
-      Resolver.bintrayRepo("hmrc", "releases"),
-      Resolver.typesafeRepo("releases")
-    )
+    commonSettings,
+    publish := {},
+    publishAndDistribute := {},
+    crossScalaVersions := Seq.empty
+  )
+  .aggregate(
+    httpVerbsPlay25,
+    httpVerbsPlay26,
+    httpVerbsPlay27
   )
   .disablePlugins(sbt.plugins.JUnitXmlReportPlugin)
+
+lazy val httpVerbsCommon = Project("http-verbs-common", file("http-verbs-common"))
+  .disablePlugins(SbtGitVersioning)
+
+lazy val httpVerbsPlay25 = Project("http-verbs-play-25", file("http-verbs-play-25"))
+  .enablePlugins(SbtAutoBuildPlugin, SbtArtifactory)
+  .settings(
+    commonSettings,
+    unmanagedSourceDirectories in Compile += (httpVerbsCommon / Compile / scalaSource).value,
+    unmanagedSourceDirectories in Test += (httpVerbsCommon / Test / scalaSource).value,
+    crossScalaVersions := Seq(scala2_11),
+    libraryDependencies ++= AppDependencies.compileCommon ++ AppDependencies.compilePlay25 ++ AppDependencies.testCommon ++ AppDependencies.testPlay25,
+    Test / fork := true // akka is not unloaded properly, which can affect other tests
+  )
+
+lazy val httpVerbsPlay26 = Project("http-verbs-play-26", file("http-verbs-play-26"))
+  .enablePlugins(SbtAutoBuildPlugin, SbtArtifactory)
+  .settings(
+    commonSettings,
+    crossScalaVersions := Seq(scala2_11, scala2_12),
+    libraryDependencies ++= AppDependencies.compileCommon ++ AppDependencies.compilePlay26 ++ AppDependencies.testCommon ++ AppDependencies.testPlay26,
+    unmanagedSourceDirectories in Compile += (httpVerbsCommon / Compile / scalaSource).value,
+    unmanagedSourceDirectories in Test += (httpVerbsCommon / Test / scalaSource).value,
+    Test / fork := true // akka is not unloaded properly, which can affect other tests
+  )
+
+lazy val httpVerbsPlay27 = Project("http-verbs-play-27", file("http-verbs-play-27"))
+  .enablePlugins(SbtAutoBuildPlugin, SbtArtifactory)
+  .settings(
+    commonSettings,
+    crossScalaVersions := Seq(scala2_12),
+    libraryDependencies ++= AppDependencies.compileCommon ++ AppDependencies.compilePlay27 ++ AppDependencies.testCommon ++ AppDependencies.testPlay27,
+    unmanagedSourceDirectories in Compile += (httpVerbsCommon / Compile / scalaSource).value,
+    unmanagedSourceDirectories in Test += (httpVerbsCommon / Test / scalaSource).value,
+    unmanagedSourceDirectories in Compile += baseDirectory.value / "../http-verbs-play-26/src/main/scala",
```

Comment:
You can do: `scalaSource in Compile := (httpVerbsPlay26 / Compile / scalaSource).value`
the tests are missing:
`scalaSource in Test := (httpVerbsPlay26 / Test / scalaSource).value`

Reply:
Change made, pushed. There was no need to add the tests as only the main code was required in the Play27 module.